sandy/core/endf6.py

```diff
@@ -1525,3 +1525,239 @@ def get_errorr(self,
+        ign : `int`, optional
+            neutron group option (default is 2, csewg 239-group structure)
+        iwt : `int`, optional
+            weight function option (default is 2, constant)
+        sigz : iterable of `float`
+            sigma zero values. The default is 1.0e10.
+
+        Returns
+        -------
+        gendf : `sandy.Gendf`
+            Gendf object
+
+        Examples
+        --------
+        Default temperature test:
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "xs", 10010)
+        >>> out = endf6.get_gendf(verbose=True)
+        moder
+        20 -21 /
+        reconr
+        -21 -22 /
+        'sandy runs njoy'/
+        125 0 0 /
+        0.005 0. /
+        0/
+        broadr
+        -21 -22 -23 /
+        125 1 0 0 0. /
+        0.005 /
+        293.6 /
+        0 /
+        groupr
+        -21 -23 0 -24 /
+        125 2 0 2 0 1 1 0 /
+        /
+        293.6/
+        10000000000.0/
+        3/
+        0/
+        0/
+        moder
+        -24 32 /
+        stop
+
+        Test various sigz:
+        >>> out = endf6.get_gendf(verbose=True, sigz=[1.0e10, 1.0e10])
+        moder
+        20 -21 /
+        reconr
+        -21 -22 /
+        'sandy runs njoy'/
+        125 0 0 /
+        0.005 0. /
+        0/
+        broadr
+        -21 -22 -23 /
+        125 1 0 0 0. /
+        0.005 /
+        293.6 /
+        0 /
+        groupr
+        -21 -23 0 -24 /
+        125 2 0 2 0 1 2 0 /
+        /
+        293.6/
+        10000000000.0 10000000000.0/
+        3/
+        0/
+        0/
+        moder
+        -24 32 /
+        stop
+
+        Test iwt:
+        >>> out = endf6.get_gendf(verbose=True, iwt=3)
+        moder
+        20 -21 /
+        reconr
+        -21 -22 /
+        'sandy runs njoy'/
+        125 0 0 /
+        0.005 0. /
+        0/
+        broadr
+        -21 -22 -23 /
+        125 1 0 0 0. /
+        0.005 /
+        293.6 /
+        0 /
+        groupr
+        -21 -23 0 -24 /
+        125 2 0 3 0 1 1 0 /
+        /
+        293.6/
+        10000000000.0/
+        3/
+        0/
+        0/
+        moder
+        -24 32 /
+        stop
+
+        Test ign:
+        >>> out = endf6.get_gendf(verbose=True, ign=3)
+        moder
+        20 -21 /
+        reconr
+        -21 -22 /
+        'sandy runs njoy'/
+        125 0 0 /
+        0.005 0. /
+        0/
+        broadr
+        -21 -22 -23 /
+        125 1 0 0 0. /
+        0.005 /
+        293.6 /
+        0 /
+        groupr
+        -21 -23 0 -24 /
+        125 3 0 2 0 1 1 0 /
+        /
+        293.6/
+        10000000000.0/
+        3/
+        0/
+        0/
+        moder
+        -24 32 /
+        stop
+
+        Test `to_file`
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "xs", 10010)
+        >>> out = endf6.get_errorr(to_file="out.gendf")
+        >>> assert os.path.isfile('out.gendf')
+
+        Error and groupr:
+        >>> out = endf6.get_gendf(verbose=True, temperature=300.0, errorr=True)
```

Comment:
This option should be removed. I expect the user to call `get_gendf` if they want a GENDF file and `get_errorr` if they want a ERRORR file. This is because in output we return only one file, not two.

In theory there should be the possibility in `get_errorr` to first pass through groupr, but then errorr should read from the gendf file as input. This will be needed to run ERRORR for nubar

Reply:
Implemented